kss-node/kss-node
# 84: Make name of styleguide.md configurable

Steps to test:

run
`./bin/kss-node --config demo/config.json`

demo styleguide should compile with new homepage.md 

in demo/config.json change config file to: (remove 'homepage' config )
```
{
  "source":       ".",
  "destination":  "../gh-pages",
  "css": [
    "public/styles.css"
  ],
  "xdemo":        1
}
```
change homepage.md to styleguide.md

run
`./bin/kss-node --config demo/config.json`

demo styleguide should compile normally
